db/write_concern.go

```diff
@@ -29,335 +28,191 @@ const (
 	tagString = "tagset"
 )
 
+// WriteConcernOptions is a shim in between write concern option configuration
+// inputs and the Go driver write concern option type, which is currently
+// opaque (GODRIVER-752).  Having the shim allows easier testing of
```

Comment:
It looks like they recently added [accessors](https://godoc.org/go.mongodb.org/mongo-driver/mongo/writeconcern#WriteConcern) to `WriteConcern` (as part of GODRIVER-844 and GODRIVER-790), so this comment could use an update.

Reply:
Or get rid of the shim type entirely and just make this code about converting directly to the driver form.   Great observation; thank you!